os/kernel/task/task_setup.c

```diff
@@ -173,7 +173,9 @@ static int task_assignpid(FAR struct tcb_s *tcb)
 #ifdef CONFIG_SCHED_CPULOAD
 			int cpuload_idx;
 			for (cpuload_idx = 0; cpuload_idx < SCHED_NCPULOAD; cpuload_idx++) {
-				g_pidhash[hash_ndx].ticks[cpuload_idx] = 0;
+				for (int i = 0; i < CONFIG_SMP_NCPUS; i++) {
```

Comment:
Instead of using variable name as i, we can name it as cpu to make it more meaningful.
You can change accordingly at all places in your pr.